connect-inject/envoy_sidecar.go

```diff
@@ -8,13 +8,23 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+type sidecarContainerCommandData struct {
+	AuthMethod      string
+	ConsulNamespace string
+}
+
 func (h *Handler) envoySidecar(pod *corev1.Pod) (corev1.Container, error) {
+	// Inputs to the sidecar template rendering
+	data := sidecarContainerCommandData{
```

Comment:
maybe call this `templateData` and then you don't need the comment

Reply:
Updated.